modular_skyrat/modules/deforest_medical_items/code/injectors.dm

```diff
@@ -0,0 +1,208 @@
+		/datum/reagent/toxin/lipolicide = 5,
+	)
+
+// Adrenaline, fills you with determination (and also stimulants)
+/obj/item/reagent_containers/hypospray/medipen/deforest/adrenaline
+	name = "adrenaline injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of reagents to intentionally give yourself fight or flight on demand."
+	base_icon_state = "adrenaline"
+	icon_state = "adrenaline"
+	list_reagents = list(
+		/datum/reagent/medicine/synaptizine = 5,
+		/datum/reagent/medicine/inaprovaline = 5,
+		/datum/reagent/determination = 10,
+		/datum/reagent/toxin/histamine = 5,
+	)
+
+// Morpital, heals a small amount of damage and kills pain for a bit
+/obj/item/reagent_containers/hypospray/medipen/deforest/morpital
+	name = "morpital regenerative stimulant injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of reagents to numb pain and repair small amounts of physical damage."
+	base_icon_state = "morpital"
+	icon_state = "morpital"
+	list_reagents = list(
+		/datum/reagent/medicine/morphine = 5,
+		/datum/reagent/medicine/omnizine/protozine = 15,
+		/datum/reagent/toxin/staminatoxin = 5,
+	)
+
+// Lipital, heals more damage than morpital but doesnt work much at higher damages
+/obj/item/reagent_containers/hypospray/medipen/deforest/lipital
+	name = "lipital regenerative stimulant injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of reagents to numb pain and repair small amounts of physical damage. \
+		Works most effectively against damaged caused by brute attacks."
+	base_icon_state = "lipital"
+	icon_state = "lipital"
+	list_reagents = list(
+		/datum/reagent/medicine/lidocaine = 5,
+		/datum/reagent/medicine/omnizine = 5,
+		/datum/reagent/medicine/c2/probital = 10,
+	)
+
+// Anti-poisoning injector, with a little bit of radiation healing as a treat
+/obj/item/reagent_containers/hypospray/medipen/deforest/meridine
+	name = "meridine antidote injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of reagents to serve as antidote to most galactic toxins. \
+		A warning sticker notes it should not be used if the patient is physically damaged, as it may cause complications."
+	base_icon_state = "meridine"
+	icon_state = "meridine"
+	list_reagents = list(
+		/datum/reagent/medicine/ammoniated_mercury = 5,
+		/datum/reagent/medicine/potass_iodide = 15,
+		/datum/reagent/nitrous_oxide = 5,
+	)
+
+// Epinephrine and helps a little bit against stuns and stamina damage
+/obj/item/reagent_containers/hypospray/medipen/deforest/synephrine
+	name = "synephrine emergency stimulant injector"
+	desc = "A Deforest branded autoinjector, loaded with a mix of reagents to stabilize critical condition and recover from stamina deficits."
+	base_icon_state = "synephrine"
+	icon_state = "synephrine"
+	list_reagents = list(
+		/datum/reagent/medicine/epinephrine = 10,
+		/datum/reagent/medicine/synaptizine = 5,
+		/datum/reagent/medicine/synaphydramine = 5,
+		/datum/reagent/nitrous_oxide = 5,
```

Comment:
I'm not quite sure what the nitrous oxide is there for, because if they're bleeding with this in them, their bleeding will get steadily worse.

Reply:
I'm not sure either actually, the n2o in both this and the next one you mention are gone